automatic/360ts/update.ps1

```diff
@@ -0,0 +1,32 @@
+
+import-module au
+
+$URL32 = 'http://int.down.360safe.com/totalsecurity/360TS_Setup.exe'
+$releases = 'https://www.360totalsecurity.com/en/features/360-total-security/'
+
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(^\s*url\s*=\s*)('.*')" = "`$1'$($Latest.URL32)'"
+      "(^\s*checksum\s*=\s*)('.*')" = "`$1'$($Latest.Checksum32)'"
+      "(^\s*checksumType\s*=\s*)('.*')" = "`$1'$($Latest.ChecksumType32)'"
+    }
+  }
+}
+
+function global:au_GetLatest {
+
```

Comment:
any reason you don't parse <https://www.360totalsecurity.com/en/download-free-antivirus/360-total-security/?offline=1> for the download link and the version from the parsed link?
that would enable you to use `-UseBasicParsing` for `Invoke-WebRequest` and be better IMO.